robottelo/ui/settings.py

```diff
@@ -13,6 +13,27 @@ def navigate_to_entity(self):
         """Navigate to Settings entity page"""
         Navigator(self.browser).go_to_settings()
 
+    def remove_parameter(self, tab_locator, param_name):
+        """Removes the value of selected parameter under settings"""
+        if self.wait_until_element(tab_locator) is None:
+            raise UINoSuchElementError(
+                "Couldn't find the tab with name: '%s'" % tab_locator)
+        self.click(tab_locator)
+
+        loc = locators['settings.edit_param']
+        if self.wait_until_element(loc % param_name) is None:
```

Comment:
same for this error handling and whole condition structure. I know that message you created is better and more readable for person who will analyze logs, but we cannot allow to expand code so much